scripts/check

```diff
@@ -1,4 +1,5 @@
-#!/bin/sh
-set -e
+#!/bin/bash
+set -Eeou pipefail
```

Comment:
Are these changes necessary? There's no pipe.

Reply:
My thinking on this is that I use this shell boilerplate everywhere as it provides a safe default for any case. In my experience scripts usually evolve and then I forget to add this and stuff breaks. So I'd prefer to use `set -Eeou pipefail` in all shell scripts throughout the codebase.

Does that make sense?

Reply:
Yeah, the reason I don't use it unless necessary is that it breaks POSIX shell compatibility, ie. it only works with bash and bash derivatives, which I tend to not install on my systems.